tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "noImplicitAny": true,
     "strictNullChecks": true,
     "sourceMap": true,
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "lib": ["es5", "dom", "es2015.core"]
```

Comment:
Do you remember why we need to add this?

Reply:
In order to use `Math.sign()`